packages/flame_splash_screen/pubspec.yaml

```diff
@@ -0,0 +1,22 @@
+name: flame_splash_screen
+description: Style your flame game with a beautiful splash screen with logo reveal. Simple to use but very customizable.
+version: 0.1.0
+homepage: https://github.com/flame-engine/flame_splash_screen
+
+environment:
+  sdk: ">=3.0.0 <4.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  meta: ^1.7.0
+
+dev_dependencies:
+  flame_lint: ^1.1.1
+  flutter_test:
+    sdk: flutter
+  mockito: ^5.0.16
+
```

Comment:
can we add everything from [here](https://raw.githubusercontent.com/flame-engine/flame/main/packages/flame_fire_atlas/pubspec.yaml)?
like funding, etc